src/unix/assets/86Box.spec

```diff
@@ -119,5 +122,5 @@ popd
 %{_bindir}/roms
 
 %changelog
-* Sat Mar 19 2022 Robert de Rooy <robert.de.rooy[AT]gmail.com> 3.3-1
-- Initial RPM release
+* Fri Apr 22 2022 Robert de Rooy <robert.de.rooy[AT]gmail.com> 3.4.1-1
```

Comment:
Would be a nice addition if this used the date global and Version. Don't know if that's possible though.

Reply:
No, unfortunately it is not.

Reply:
what some other projects do, is to run some script before release on the project that updates all the version and date references everywhere.